tests/security/tpm2/tpm2_tools/tpm2_tools_auth.pm

```diff
@@ -23,45 +23,45 @@ sub run {
     my $key_pub = "key.pub";
     my $key_name = "key.name";
     my $ket_ctx = "key.ctx";
+
+    my $tpm_suffix = '';
+    $tpm_suffix .= '-T tabrmd' if (get_var('QEMUTPM', 0) != 1 || get_var('QEMUTPM_VER', '') ne '2.0');
```

Comment:
`$tmp_suffix` is already an empty string in the above line. so I think just using `$tmp_suffix = '-T tabrmd'` should be ok.